gce/worker/prepare.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -x
+
+if [ -e "/opt/app/environment.yml" ]; then
+    echo "environment.yml found. Installing packages"
+    /opt/conda/bin/conda env update -f /opt/app/environment.yml
+else
+    echo "no environment.yml"
+fi
+
+if [ "$EXTRA_CONDA_PACKAGES" ]; then
+    echo "EXTRA_CONDA_PACKAGES environment variable found.  Installing."
+    /opt/conda/bin/conda install $EXTRA_CONDA_PACKAGES
+fi
+
+if [ "$EXTRA_PIP_PACKAGES" ]; then
+    echo "EXTRA_PIP_PACKAGES environment variable found.  Installing".
```

Comment:
Is this the recommended way of specifying custom packages for workers now? As apposed to specifying them in the `.daskernetes.yaml`. 

cc @atlehr

Reply:
You can specify a complete custom docker image, or you can specify environment variables like `EXTRA_PIP_PACKAGES`.  Those environment variables get picked up when the docker container starts and runs this script.  This is how it has always worked.

Note that the `.daskernetes.yaml` file is no longer used.  Instead you point to a file like worker-template.yaml.  This is specifyable when you create a cluster.  See http://daskernetes.readthedocs.io/en/latest/